modules/search/src/test/java/org/opencastproject/matterhorn/search/impl/ElasticsearchUtils.java

```diff
@@ -42,29 +41,23 @@ private ElasticsearchUtils() {
 
   /**
    * Creates an elastic search index configuration inside the given directory by loading the relevant configuration
-   * files from the bundle. The final location will be <code>homeDirectory/etc/index</code>.
+   * files from the bundle. The final location will be <code>configDirectory/etc/index</code>.
    * 
-   * @param homeDirectory
+   * @param configDirectory
    *          the configuration directory
-   * @param index
-   *          the index name
    * @throws IOException
    *           if creating the configuration fails
    */
-  public static void createIndexConfigurationAt(File homeDirectory, String index) throws IOException {
+  public static void createIndexConfigurationAt(File configDirectory) throws IOException {
 
     // Load the index configuration and move it into place
-    File configurationRoot = new File(PathSupport.concat(new String[] { homeDirectory.getAbsolutePath(), index }));
-    FileUtils.deleteQuietly(configurationRoot);
-    if (!configurationRoot.mkdirs())
-      throw new IOException("Error creating " + configurationRoot);
-
-    String[] files = new String[] { "content-mapping.json", "names.txt", "settings.yml", "version-mapping.json" };
+    String[] files = new String[] { "content-mapping.json", "elasticsearch.yml", "version-mapping.json" };
```

Comment:
content-mapping.json?

Reply:
I'm not sure what your question here is.

Reply:
The file does not exist. But anyway, it is not this PR that "introduced" it.

Reply:
Just realised that this is test code and there indeed is a content-mapping.json in the test resources... sorry for the noise.

Reply:
Yes, it's a bit confusing that the utils classes are only used for tests.